File: 02-03-edittext/java-02-03-edittext/app/src/main/java/com/example/edittext/MainActivity.java
package com.example.edittext;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView textView;
    private EditText editText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = findViewById(R.id.text);
        editText = findViewById(R.id.editText);

        Button button = findViewById(R.id.button);

        button.setOnClickListener(v -> textView.setText(editText.getText().toString()));
    }
}

File: 02-03-edittext/kotlin-02-03-edittext/app/src/main/java/com/example/edittext/MainActivity.kt
package com.example.edittext

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private var textView: TextView? = null
    private var editText: EditText? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        textView = findViewById(R.id.text)
        editText = findViewById(R.id.editText)

        val button = findViewById<Button>(R.id.button)

        button.setOnClickListener {
            textView?.text = editText?.text.toString()
        }
    }
}
